homeassistant/components/zwave_js/climate.py

```diff
@@ -421,7 +419,7 @@ def extra_state_attributes(self) -> dict[str, str] | None:
     @property
     def min_temp(self) -> float:
         """Return the minimum temperature."""
-        min_temp = DEFAULT_MIN_TEMP
+        min_temp = 0.0  # Not using DEFAULT_MIN_TEMP to allow wider range
```

Comment:
Maybe make it `-50` to support freezers?

Reply:
I was just speaking hypothetically. Not sure if that will ever be an issue, and if it is, we could fix it then. Because right now, 0° seems more reasonable for those users with V1 thermostats.

Reply:
Ok.